usecases/objects/references_delete.go

```diff
@@ -166,23 +167,24 @@ func (req *DeleteReferenceInput) validateSchema(class *models.Class) error {
 
 // removeReference removes ref from object obj with property prop.
 // It returns ok (removal took place) and an error message
-func removeReference(obj *models.Object, prop string, ref *models.SingleRef) (ok bool, errmsg string) {
+func removeReference(obj *models.Object, prop string, remove *models.SingleRef) (ok bool, errmsg string) {
 	properties := obj.Properties.(map[string]interface{})
 	if properties == nil || properties[prop] == nil {
 		return false, ""
 	}
 
 	refs, ok := properties[prop].(models.MultipleRef)
 	if !ok {
-		return false, "source list is not well formed"
+		return false, fmt.Sprintf("property %q with type %T is not a valid cross-reference", prop, refs)
```

Comment:
I would go with `%s` instead of `%q` here, bc it's then easier to write checks bc the error message would not contain `"` signs then